index.Rmd

```diff
@@ -304,101 +303,127 @@ ggplot(us_data, aes(x = med_income, y = med_prop_val)) +
 
 Learning how to describe data visualizations is a very useful skill. We will provide descriptions for you in this course (as we did above) until we get to Chapter 4, 
 which focuses on data visualization. Then, we will explicitly teach you how to do this yourself, and how to not over-state or over-interpret the results 
 from a visualization.
 
-### Coloring points by group
-Now we'll move onto the second part of our exploratory data analysis question: when considering the relationship between median household income and median property 
-value, is there a pattern in which party each state voted for in the 2016 US election? One common way to explore this is to colour the data points on the 
-scatter plot we have already created by group/category. For example,
-given that we have the party each state voted for in the 2016 US Presidential election in the column named `party`, we can colour the points in our previous 
-scatter plot to represent who each stated voted for. 
-
-To do this we modify our scatter plot code above. Specifically, we will add an argument to the `aes` function, specifying that the points should be coloured by the `party` column:
+### Changing the units
 
-```{r relevel factors for colour, include=FALSE}
-us_data <- us_data %>% mutate(party = factor(party, levels = c("Republican", "Not Applicable", "Democratic")))
+```{r}
+english_mother_tongue <- can_lang %>% 
+  filter(language == "English") %>% 
+  pull(mother_tongue)
+census_popn <- 35151728
 ```
 
-<center>
-```{r scatter_colour_by_party, fig.width=5.75, fig.height=4, warning=FALSE}
-ggplot(us_data, aes(x = med_income, y = med_prop_val, color = party)) +
-  geom_point() +
-  xlab("Income (USD)") +
-  ylab("Median property value (USD)")
-```
-</center>
+What does it mean that `r format(english_mother_tongue, scientific = FALSE, big.mark = ",")` people reported that their mother tongue was English in the 2016 Canadian census? To really understand this number, we need context, in particular, how many people are there were in Canada when this data was collected? From the 2016 Canadian census profile we can see that the population was reported to be `r format(census_popn, scientific = FALSE, big.mark = ",")` people. The absolute count of the number of people who report that English is their mother tongue is much more meaningful when we report it in this context. We can even go a step further and transform this absolute count to a relative frequency, or proportion, so that we can represent this as a single meaningful number in our data visualizations. We can do this by dividing the count of the number of people reporting a given language as their mother tongue by the count of the number of peole who live in Canada. For example, the proportion of people who eported that their mother tongue was English in the 2016 Canadian census was `r format(round(english_mother_tongue/census_popn, 2), scientific = FALSE, big.mark = ",")`.
```

Comment:
```suggestion
What does it mean that `r format(english_mother_tongue, scientific = FALSE, big.mark = ",")` people reported that their mother tongue was English in the 2016 Canadian census? To really understand this number, we need context. In particular, how many people were in Canada when this data was collected? From the 2016 Canadian census profile, we can see that the population was reported to be `r format(census_popn, scientific = FALSE, big.mark = ",")` people. The absolute count of the number of people who report that English is their mother tongue is much more meaningful when we report it in this context. We can even go a step further and transform this absolute count to a relative frequency, or proportion, so that we can represent this as a single meaningful number in our data visualizations. We can do this by dividing the number of people reporting a given language as their mother tongue by the number of people who live in Canada. For example, the proportion of people who reported that their mother tongue was English in the 2016 Canadian census was `r format(round(english_mother_tongue/census_popn, 2), scientific = FALSE, big.mark = ",")`.
```

Reply:
Accepted this, but will modify further as I have no idea why I thought I should call it an absolute count as opposed to a count.